package.json

```diff
@@ -42,6 +42,8 @@
     "@ts-stack/markdown": "^1.4.0",
     "@types/dompurify": "^3.0.2",
     "@types/enzyme": "^3.10.13",
+    "@types/react": "^17",
+    "@types/react-dom": "^17",
```

Comment:
These two are listed under `resolutions` instead of `dependencies`. I don't recall why, but I recall it was on purpose.

Is it possible adding these 2 deps would do something unexpected?

Reply:
sorry, not following. where are they listed under resolutions?